README.rst

```diff
@@ -32,10 +32,108 @@ Issue for its creation at https://github.com/unfoldingWord-dev/door43.org/issues
 tX Pipeline
 ===========
 
-1. webhook
-2. job_request
-3. job_handle
-4. [CONVERTER]
-5. callback
-6. door43_deploy
-7. d43_catalog
+1. Gogs
+2. Webhook
+3. Request Job
+4. Start Job
+5. [CONVERTER]
+6. Callback
+7. Door43 Deploy
+
+Definitions
+===========
+The following placeholders are used in examples in this document:
+
+* <repo> - the machine name of a Gog's repo. This is used in the URL for the repo, such as en-obs
+* <user> - the user or organization that the Gog's repo belongs to, such as richmahn (user) or door43 (org)
+* <commit> - The 10 character hash string that represents the commit (revision) that is being processed
+
+How it Works
+============
+
+Request Conversion Job
+----------------------
+
+Using the Pipeline and the corresponding numbers above, this describes each part of the pipeline and how each are integrated, both with each other as well as the AWS Services that are used.
+
+NOTE: This gives URLs and bucket names for development. For production, remove the `test-` prefix from domain or bucket name
+
+1. Gogs (Git website)
+
+When a repository is updated on `Gogs <https://test.door43.org:3000>`_, the commit triggers all webhooks in the repo's settings to be triggered. One of those webhooks, which our copy of Gogs sets up automatically for every new repo, is a call to https://test-api.door43.org/client/webhook (API Gateway -> Lambda function).
```

Comment:
"triggers...to be triggered" make no sense. Would be better if "to be triggered" was removed.